pkg/defaults/defaults.go

```diff
@@ -530,6 +530,9 @@ const (
 
 	// Enable BGP control plane features.
 	EnableBGPControlPlane = false
+
+	// EnableK8sNetworkPolicy default value is true, cilium agent need to support K8s NetworkPolicy by default.
```

Comment:
s/need/needs/, but then we don't need to give so many precisions (or to provide the default value again) here.
```suggestion
	// EnableK8sNetworkPolicy enables support for K8s NetworkPolicy.
```